solidity/ecdsa/contracts/WalletRegistry.sol

```diff
@@ -76,10 +76,16 @@ contract WalletRegistry is
     ///         other member that will call the DKG approve function.
     uint256 public dkgResultSubmissionGas = 275000;
 
-    // @notice Gas meant to balance the DKG result approval's overall cost. Can
-    //         be updated by the governace based on the current market conditions.
+    /// @notice Gas is meant to balance the DKG result approval's overall cost.
```

Comment:
Should it be "Gas is meant to..." or "Gas meant to"? My understanding is that the second is correct, given that we are explaining what `dkgResultApprovalGasOffset` is for.

```suggestion
    /// @notice Gas meant to balance the DKG result approval's overall cost.
```